src/Symfony/Component/Security/Core/Tests/Authorization/Voter/AbstractVoterTest.php

```diff
@@ -65,16 +65,6 @@ public function testVoteLegacy(array $attributes, $expectedVote, $object, $messa
 
         $this->assertEquals($expectedVote, $voter->vote($this->token, $object, $attributes), $message);
     }
-
-    /**
-     * @group legacy
-     * @expectedException \BadMethodCallException
-     */
-    public function testNoOverriddenMethodsThrowsException()
-    {
-        $voter = new AbstractVoterTest_NothingImplementedVoter();
-        $voter->vote($this->token, new \stdClass(), array('EDIT'));
-    }
```

Comment:
Never mind, I forgot to fix the `getSupportedClasses()` method of the `AbstractVoterTest_NothingImplementedVoter` class.